### DISCUSSION TITLE
Maximum and Minimum number of terms per post

### DISCUSSION THREAD

**Discussion by stevejburge** [publishpress/publishpress-taxonomies#1719] [Feature Requests]
Related to this: https://github.com/TaxoPress/TaxoPress/issues/1676

We could add a setting for the maximum and Minimum number of terms per post

**Comment by ojopaul** [comment]
@stevejburge Where are we putting this settings and also note, this will require lot of thinking, trial and error work since there's no custom way of doing it especially in showing the error message on the screen and Block editor integration.

The error will probably be similar to Checklists taxonomy limit error and blocking publishing but just that this covers all terms in taxonomy especially in block editor

**Comment by stevejburge** [comment]
@ojopaul Maybe we need our own metaboxes for terms?
